websites/S/Stadia/presence.ts

```diff
@@ -41,4 +41,4 @@ presence.on("UpdateData", async () => {
   } else {
     presence.setActivity(presenceData);
   }
-});
+});
```

Comment:
revert

Reply:
how? I tried.

Reply:
Revert changes in VSC for example 🤷 then do commit

Reply:
I did that but it still shows that